oat-sa/tao-core
# [ACTP-311] Login Page - Fix Login Component and Focus Style

Related to : https://oat-sa.atlassian.net/browse/ACTP-311
 
Refactor login form:

- introduce disableAutofocus login configuration option
- add class to login form entrypoint links
 
#### How to test
 
- Prepare an instance of TAO with taoAct extension
- Change `disableAutofocus` to `true` in config/tao/login.conf.php
- Navigate to login page
- Check that there is no autofocus on login field and screen reader reads page from the top
- Check that screen reader can recognize show password controls
- check focus styles of `login` and guest `access buttons`
   
Companion PR :
 - [ ] https://github.com/oat-sa/tao-core-ui-fe/pull/99
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1308
